src/markdown-engine-config.ts

```diff
@@ -58,4 +64,5 @@ export const defaultMarkdownEngineConfig: MarkdownEngineConfig = {
   pandocArguments: [],
   latexEngine: "pdflatex",
   enableScriptExecution: false,
+  enableHtml5Embed: false,
 };
```

Comment:
You might also want to add default configs for `html5EmbedUseImageSyntax`, `html5EmbedUseImageSyntax`, `html5EmbedIsAllowedHttp`, `html5EmbedAudioAttributes` and `html5EmbedVideoAttributes`. Right now you only added `enableHtml5Embed`. 
What's more, please change `enableHtml5Embed` to `enableHTML5Embed`, because I personally think `HTML` is better than `Html` in a variable name.  Thank you!

Reply:
Besides, please include the version of html5-embed in `dependencies/markdown-it/README.md`. Thanks!

Reply:
Done!